includes/data-stores/class-wc-admin-reports-orders-data-store.php

```diff
@@ -151,8 +161,8 @@ public function get_data( $query_args ) {
 			'page'             => 1,
 			'order'            => 'DESC',
 			'orderby'          => 'date_created',
-			'before'           => WC_Admin_Reports_Interval::default_before(),
-			'after'            => WC_Admin_Reports_Interval::default_after(),
+			'before'           => '',
```

Comment:
I like the idea of removing the default date limitations.  Just want to make sure it matches up with what is done in https://github.com/woocommerce/woocommerce-admin/pull/2636.

Reply:
There might be an errant change in #2636, see: https://github.com/woocommerce/woocommerce-admin/pull/2636/files#r304572283.

Reply:
I think this PR is going to do the trick without implementing #2636.